src/server/server_family.cc

```diff
@@ -3246,11 +3249,78 @@ string ServerFamily::FormatInfoMetrics(const Metrics& m, std::string_view sectio
            "errors]");
   }
 
+#ifdef WITH_SEARCH
   if (should_enter("SEARCH", true)) {
     append("search_memory", m.search_stats.used_memory);
     append("search_num_indices", m.search_stats.num_indices);
     append("search_num_entries", m.search_stats.num_entries);
+
+    // Developer-visible fields - controlled by search_info_developer_visible flag
+    // These fields provide detailed internal statistics useful for debugging and monitoring
+    bool show_developer_fields = absl::GetFlag(FLAGS_search_info_developer_visible);
+
+    if (show_developer_fields) {
+      // Queue and worker pool statistics
```

Comment:
Do we really need this? I doubt clients rely on this - those are some internal valkey stats

Reply:
Skip some of them or some parts if possible 🙂